src/libraries/Microsoft.Extensions.Http/src/ExpiredHandlerTrackingEntry.cs

```diff
@@ -25,10 +25,10 @@ public ExpiredHandlerTrackingEntry(ActiveHandlerTrackingEntry other)
 
         public bool CanDispose => !_livenessTracker.IsAlive;
 
-        public HttpMessageHandler InnerHandler { get; }
```

Comment:
As we discussed before, InnerHandler here cannot be null, so I think instead of making it nullable, we can just do `InnerHandler = other.Handler.InnerHandler!;` in the constructor above?